test/cmdlineTests/yul_ethdebug/output

```diff
@@ -0,0 +1,17 @@
+======= Debug Data (ethdebug/format/object) =======
+{"sources":["yul_ethdebug/input.yul"]}
+
+======= yul_ethdebug/input.yul (EVM) =======
+
+Pretty printed source:
+object "object" {
```

Comment:
`/// ethdebug: enabled` missing - I wanted to later only parse ethdebug attributes, if it was enabled. Need to check whether this information was transported correctly.